project/templates/configs/springboard.ini

```diff
@@ -1,6 +1,8 @@
 [app:springboardfrontend]
 use = egg:springboard_{{ app_type }}
-unicore.content_repos = unicore_frontend_{{app_type}}_{{country}}
+unicore.content_repo_urls = {{ucd_host}}/repos/{{repo_name}}.json
+
+es.host = {{es_host}}
```

Comment:
Not yet. I tested it locally and it gave me some problems. So let's leave it for now.